scripts/eachdist.py

```diff
@@ -503,6 +515,117 @@ def lint_args(args):
     )
 
 
+def update_changelog(path, version, new_entry):
+    unreleased_changes = 0
+    try:
+        with open(path) as changelog:
+            text = changelog.read()
+            if version in text:
```

Comment:
We should look for `## 1.2.3` to be more specific.

Reply:
Agreed, checking for `## Version 1.2.3` now